2025-07-30 23:10:09,994 - task_service - INFO - ================================================================================
2025-07-30 23:10:09,994 - task_service - INFO - Starting Task Consumer at 2025-07-30 23:10:09.994659
2025-07-30 23:10:09,994 - task_service - INFO - ================================================================================
2025-07-30 23:17:59,950 - task_service - INFO - Creating new task - ID: fd86f35e-3533-48e4-b46b-1b6d875bd74c
2025-07-30 23:17:59,951 - task_service - INFO - Task Name: un ticket
2025-07-30 23:17:59,951 - task_service - INFO - Content: con su descripci√≥n
2025-07-30 23:17:59,951 - task_service - INFO - Assignee: y es asignado a
2025-07-30 23:17:59,951 - task_service - INFO - Sending task fd86f35e-3533-48e4-b46b-1b6d875bd74c to message queue
2025-07-30 23:17:59,965 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-07-30 23:17:59,966 - pika.adapters.utils.io_services_utils - ERROR - Socket failed to connect: <socket.socket fd=14, family=30, type=1, proto=6, laddr=('::1', 64128, 0, 0)>; error=61 (Connection refused)
2025-07-30 23:17:59,966 - pika.adapters.utils.connection_workflow - ERROR - TCP Connection attempt failed: ConnectionRefusedError(61, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-07-30 23:17:59,966 - pika.adapters.utils.connection_workflow - ERROR - AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(61, 'Connection refused')
2025-07-30 23:17:59,967 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-07-30 23:17:59,967 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64129), raddr=('127.0.0.1', 5672)>
2025-07-30 23:17:59,968 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-07-30 23:17:59,977 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:17:59,977 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:17:59,977 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:17:59,978 - pika.adapters.blocking_connection - INFO - Created channel=1
2025-07-30 23:17:59,979 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2025-07-30 23:17:59,980 - pika.channel - INFO - Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:17:59,981 - pika.channel - INFO - Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1046b5550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:17:59,981 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2025-07-30 23:17:59,982 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64129), raddr=('127.0.0.1', 5672)>
2025-07-30 23:17:59,982 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64129), raddr=('127.0.0.1', 5672)>
2025-07-30 23:17:59,982 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64129), raddr=('127.0.0.1', 5672)>
2025-07-30 23:17:59,983 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:17:59,983 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:17:59,984 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64129), raddr=('127.0.0.1', 5672)>
2025-07-30 23:17:59,985 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2025-07-30 23:17:59,986 - task_service - INFO - Message for task fd86f35e-3533-48e4-b46b-1b6d875bd74c sent to queue successfully
2025-07-30 23:17:59,986 - task_service - INFO - Task fd86f35e-3533-48e4-b46b-1b6d875bd74c created successfully
2025-07-30 23:19:18,690 - task_service - INFO - ================================================================================
2025-07-30 23:19:18,690 - task_service - INFO - Starting Task Consumer at 2025-07-30 23:19:18.690434
2025-07-30 23:19:18,690 - task_service - INFO - ================================================================================
2025-07-30 23:19:31,581 - task_service - INFO - Creating new task - ID: b5dd3e54-2e18-4de1-95fe-e0fb3df56c15
2025-07-30 23:19:31,581 - task_service - INFO - Task Name: sarasa
2025-07-30 23:19:31,581 - task_service - INFO - Content: ceresa
2025-07-30 23:19:31,581 - task_service - INFO - Assignee: zaragosa
2025-07-30 23:19:31,581 - task_service - INFO - Sending task b5dd3e54-2e18-4de1-95fe-e0fb3df56c15 to message queue
2025-07-30 23:19:31,591 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-07-30 23:19:31,591 - pika.adapters.utils.io_services_utils - ERROR - Socket failed to connect: <socket.socket fd=14, family=30, type=1, proto=6, laddr=('::1', 64244, 0, 0)>; error=61 (Connection refused)
2025-07-30 23:19:31,592 - pika.adapters.utils.connection_workflow - ERROR - TCP Connection attempt failed: ConnectionRefusedError(61, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-07-30 23:19:31,592 - pika.adapters.utils.connection_workflow - ERROR - AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(61, 'Connection refused')
2025-07-30 23:19:31,592 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-07-30 23:19:31,593 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64245), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:31,593 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-07-30 23:19:31,598 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:19:31,599 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:19:31,599 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:19:31,599 - pika.adapters.blocking_connection - INFO - Created channel=1
2025-07-30 23:19:31,600 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2025-07-30 23:19:31,600 - pika.channel - INFO - Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:19:31,601 - pika.channel - INFO - Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077b63c0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:19:31,601 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2025-07-30 23:19:31,602 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64245), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:31,602 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64245), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:31,602 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64245), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:31,602 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:19:31,603 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:19:31,603 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64245), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:31,603 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2025-07-30 23:19:31,603 - task_service - INFO - Message for task b5dd3e54-2e18-4de1-95fe-e0fb3df56c15 sent to queue successfully
2025-07-30 23:19:31,603 - task_service - INFO - Task b5dd3e54-2e18-4de1-95fe-e0fb3df56c15 created successfully
2025-07-30 23:19:36,696 - task_service - INFO - Deleting task - ID: b5dd3e54-2e18-4de1-95fe-e0fb3df56c15
2025-07-30 23:19:36,696 - task_service - INFO - Task Name: sarasa
2025-07-30 23:19:36,696 - task_service - INFO - Assignee: zaragosa
2025-07-30 23:19:36,696 - task_service - INFO - Sending delete message for task b5dd3e54-2e18-4de1-95fe-e0fb3df56c15 to message queue
2025-07-30 23:19:36,698 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-07-30 23:19:36,699 - pika.adapters.utils.io_services_utils - ERROR - Socket failed to connect: <socket.socket fd=14, family=30, type=1, proto=6, laddr=('::1', 64255, 0, 0)>; error=61 (Connection refused)
2025-07-30 23:19:36,699 - pika.adapters.utils.connection_workflow - ERROR - TCP Connection attempt failed: ConnectionRefusedError(61, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-07-30 23:19:36,699 - pika.adapters.utils.connection_workflow - ERROR - AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(61, 'Connection refused')
2025-07-30 23:19:36,699 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-07-30 23:19:36,699 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64256), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:36,700 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-07-30 23:19:36,708 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:19:36,708 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:19:36,708 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:19:36,709 - pika.adapters.blocking_connection - INFO - Created channel=1
2025-07-30 23:19:36,710 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2025-07-30 23:19:36,710 - pika.channel - INFO - Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:19:36,711 - pika.channel - INFO - Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x10775b9d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:19:36,711 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2025-07-30 23:19:36,711 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64256), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:36,711 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64256), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:36,712 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64256), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:36,712 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:19:36,712 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:19:36,712 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 64256), raddr=('127.0.0.1', 5672)>
2025-07-30 23:19:36,712 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2025-07-30 23:19:36,712 - task_service - INFO - Delete message for task b5dd3e54-2e18-4de1-95fe-e0fb3df56c15 sent to queue successfully
2025-07-30 23:19:36,712 - task_service - INFO - Task b5dd3e54-2e18-4de1-95fe-e0fb3df56c15 deleted from memory
2025-07-30 23:48:27,630 - task_service - INFO - Creating new task - ID: b650a2e4-3589-434f-89eb-b3e91e5456d3
2025-07-30 23:48:27,630 - task_service - INFO - Task Name: hola
2025-07-30 23:48:27,631 - task_service - INFO - Content: mi nombre es 
2025-07-30 23:48:27,631 - task_service - INFO - Assignee: manola
2025-07-30 23:48:27,631 - task_service - INFO - Sending task b650a2e4-3589-434f-89eb-b3e91e5456d3 to message queue
2025-07-30 23:48:27,633 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-07-30 23:48:27,634 - pika.adapters.utils.io_services_utils - ERROR - Socket failed to connect: <socket.socket fd=14, family=30, type=1, proto=6, laddr=('::1', 51368, 0, 0)>; error=61 (Connection refused)
2025-07-30 23:48:27,634 - pika.adapters.utils.connection_workflow - ERROR - TCP Connection attempt failed: ConnectionRefusedError(61, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-07-30 23:48:27,635 - pika.adapters.utils.connection_workflow - ERROR - AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(61, 'Connection refused')
2025-07-30 23:48:27,635 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-07-30 23:48:27,635 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 51369), raddr=('127.0.0.1', 5672)>
2025-07-30 23:48:27,636 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-07-30 23:48:27,649 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:48:27,649 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:48:27,650 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-30 23:48:27,654 - pika.adapters.blocking_connection - INFO - Created channel=1
2025-07-30 23:48:27,656 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2025-07-30 23:48:27,658 - pika.channel - INFO - Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:48:27,660 - pika.channel - INFO - Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x107759f90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-30 23:48:27,664 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2025-07-30 23:48:27,664 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 51369), raddr=('127.0.0.1', 5672)>
2025-07-30 23:48:27,664 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 51369), raddr=('127.0.0.1', 5672)>
2025-07-30 23:48:27,664 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 51369), raddr=('127.0.0.1', 5672)>
2025-07-30 23:48:27,664 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:48:27,664 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-30 23:48:27,665 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 51369), raddr=('127.0.0.1', 5672)>
2025-07-30 23:48:27,665 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2025-07-30 23:48:27,665 - task_service - INFO - Message for task b650a2e4-3589-434f-89eb-b3e91e5456d3 sent to queue successfully
2025-07-30 23:48:27,665 - task_service - INFO - Task b650a2e4-3589-434f-89eb-b3e91e5456d3 created successfully
2025-07-31 00:11:42,701 - task_service - INFO - Creating new task - ID: 392da3f4-f891-4ea0-b30d-a5171a743288
2025-07-31 00:11:42,703 - task_service - INFO - Task Name: hola 
2025-07-31 00:11:42,703 - task_service - INFO - Content: me lalamo
2025-07-31 00:11:42,703 - task_service - INFO - Assignee: martin
2025-07-31 00:11:42,703 - task_service - INFO - Sending task 392da3f4-f891-4ea0-b30d-a5171a743288 to message queue
2025-07-31 00:11:42,706 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-07-31 00:11:42,706 - pika.adapters.utils.io_services_utils - ERROR - Socket failed to connect: <socket.socket fd=14, family=30, type=1, proto=6, laddr=('::1', 53836, 0, 0)>; error=61 (Connection refused)
2025-07-31 00:11:42,706 - pika.adapters.utils.connection_workflow - ERROR - TCP Connection attempt failed: ConnectionRefusedError(61, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-07-31 00:11:42,707 - pika.adapters.utils.connection_workflow - ERROR - AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(61, 'Connection refused')
2025-07-31 00:11:42,707 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-07-31 00:11:42,707 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 53837), raddr=('127.0.0.1', 5672)>
2025-07-31 00:11:42,708 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-07-31 00:11:42,717 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-31 00:11:42,717 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-31 00:11:42,717 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-31 00:11:42,718 - pika.adapters.blocking_connection - INFO - Created channel=1
2025-07-31 00:11:42,719 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2025-07-31 00:11:42,720 - pika.channel - INFO - Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-31 00:11:42,720 - pika.channel - INFO - Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x1077f89d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-31 00:11:42,721 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2025-07-31 00:11:42,721 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 53837), raddr=('127.0.0.1', 5672)>
2025-07-31 00:11:42,721 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 53837), raddr=('127.0.0.1', 5672)>
2025-07-31 00:11:42,721 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 53837), raddr=('127.0.0.1', 5672)>
2025-07-31 00:11:42,721 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-31 00:11:42,722 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-31 00:11:42,722 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 53837), raddr=('127.0.0.1', 5672)>
2025-07-31 00:11:42,722 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2025-07-31 00:11:42,722 - task_service - INFO - Message for task 392da3f4-f891-4ea0-b30d-a5171a743288 sent to queue successfully
2025-07-31 00:11:42,722 - task_service - INFO - Task 392da3f4-f891-4ea0-b30d-a5171a743288 created successfully
2025-07-31 21:37:49,989 - task_service - INFO - ================================================================================
2025-07-31 21:37:49,990 - task_service - INFO - Starting Task Consumer at 2025-07-31 21:37:49.990068
2025-07-31 21:37:49,990 - task_service - INFO - ================================================================================
2025-07-31 21:38:15,751 - task_service - INFO - ================================================================================
2025-07-31 21:38:15,751 - task_service - INFO - Starting Task Consumer at 2025-07-31 21:38:15.751505
2025-07-31 21:38:15,751 - task_service - INFO - ================================================================================
2025-07-31 21:38:15,752 - fastapi - ERROR - Form data requires "python-multipart" to be installed. 
You can install "python-multipart" with: 

pip install python-multipart

2025-07-31 21:39:56,753 - task_service - INFO - ================================================================================
2025-07-31 21:39:56,753 - task_service - INFO - Starting Task Consumer at 2025-07-31 21:39:56.753281
2025-07-31 21:39:56,753 - task_service - INFO - ================================================================================
2025-07-31 21:39:56,754 - fastapi - ERROR - Form data requires "python-multipart" to be installed. 
You can install "python-multipart" with: 

pip install python-multipart

2025-07-31 21:40:13,064 - task_service - INFO - ================================================================================
2025-07-31 21:40:13,064 - task_service - INFO - Starting Task Consumer at 2025-07-31 21:40:13.064426
2025-07-31 21:40:13,064 - task_service - INFO - ================================================================================
2025-07-31 21:41:42,892 - task_service - INFO - ================================================================================
2025-07-31 21:41:42,892 - task_service - INFO - Starting Task Consumer at 2025-07-31 21:41:42.892679
2025-07-31 21:41:42,892 - task_service - INFO - ================================================================================
2025-07-31 21:44:20,683 - task_service - INFO - Creating new task - ID: c972d1ac-c444-4ad7-90a9-30d3c4be217c
2025-07-31 21:44:20,684 - task_service - INFO - Task Name: asdf
2025-07-31 21:44:20,684 - task_service - INFO - Content: asdf
2025-07-31 21:44:20,684 - task_service - INFO - Assignee: sdf
2025-07-31 21:44:20,684 - task_service - INFO - Sending task c972d1ac-c444-4ad7-90a9-30d3c4be217c to message queue
2025-07-31 21:44:20,694 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-07-31 21:44:20,694 - pika.adapters.utils.io_services_utils - ERROR - Socket failed to connect: <socket.socket fd=14, family=30, type=1, proto=6, laddr=('::1', 60495, 0, 0)>; error=61 (Connection refused)
2025-07-31 21:44:20,694 - pika.adapters.utils.connection_workflow - ERROR - TCP Connection attempt failed: ConnectionRefusedError(61, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-07-31 21:44:20,694 - pika.adapters.utils.connection_workflow - ERROR - AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(61, 'Connection refused')
2025-07-31 21:44:20,695 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-07-31 21:44:20,695 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 60496), raddr=('127.0.0.1', 5672)>
2025-07-31 21:44:20,696 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-07-31 21:44:20,705 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-31 21:44:20,705 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-31 21:44:20,705 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-07-31 21:44:20,706 - pika.adapters.blocking_connection - INFO - Created channel=1
2025-07-31 21:44:20,708 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2025-07-31 21:44:20,708 - pika.channel - INFO - Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-31 21:44:20,709 - pika.channel - INFO - Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x103081550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2025-07-31 21:44:20,709 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2025-07-31 21:44:20,710 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 60496), raddr=('127.0.0.1', 5672)>
2025-07-31 21:44:20,710 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 60496), raddr=('127.0.0.1', 5672)>
2025-07-31 21:44:20,710 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 60496), raddr=('127.0.0.1', 5672)>
2025-07-31 21:44:20,710 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-31 21:44:20,710 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2025-07-31 21:44:20,711 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=14, family=2, type=1, proto=6, laddr=('127.0.0.1', 60496), raddr=('127.0.0.1', 5672)>
2025-07-31 21:44:20,711 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2025-07-31 21:44:20,711 - task_service - INFO - Message for task c972d1ac-c444-4ad7-90a9-30d3c4be217c sent to queue successfully
2025-07-31 21:44:20,711 - task_service - INFO - Task c972d1ac-c444-4ad7-90a9-30d3c4be217c created successfully
